app/views/stories/feeds/show.json.jbuilder

```diff
@@ -17,6 +17,14 @@ json.array!(@stories) do |article|
     json.organization article.cached_organization.as_json["table"]
   end
 
+  if article.main_image?
+    json.main_image cloud_cover_url(article.main_image)
+  else
+    json.main_image nil
+  end
+
+  # article.main_image? ? cloud_cover_url(article.main_image) : nil
```

Comment:
Maybe forgot to take this line away after using it? No need to keep it right?  🙂

Reply:
Whoops: thanks!